Assets/MixedRealityToolkit.Providers/XRSDK/Controllers/GenericXRSDKController.cs

```diff
@@ -45,12 +45,6 @@ public GenericXRSDKController(TrackingState trackingState, Handedness controller
         /// <inheritdoc />
         public override MixedRealityInteractionMapping[] DefaultRightHandedInteractions => DefaultInteractions;
 
-        /// <inheritdoc />
-        public override void SetupDefaultInteractions(Handedness controllerHandedness)
```

Comment:
Should this be marked obsolete rather than removed?

Reply:
is this still experimental? if so we don't have to mark it as obsolete i think

Reply:
It's an override, which should have been removed in #7374 but I missed because I was working on 2018.4.